peachjam/templates/peachjam/document_access_group_list.html

```diff
@@ -0,0 +1,31 @@
+{% extends "user_account/layout.html" %}
+{% load i18n %}
+{% block title %}
+  {% trans 'Document access groups' %}
+{% endblock %}
+{% block account-tabs %}
+  {% include 'user_account/_tabs.html' with active='doc_access_groups' %}
+{% endblock %}
+{% block account-content %}
+  <div class="container mt-4">
+    <h2>{% trans 'Document access groups' %}</h2>
+    {% blocktrans trimmed %}
+    <p>Some documents are only visible to certain users. These groups control which restricted documents you have access to.</p>
+    {% endblocktrans %}
+    <div class="my-3">
+      {% if groups|length %}
+        <ul>
+          {% for group in groups %}
+            <li>
+              <a href="{% url 'document_access_group_detail' group.pk %}">{{ group.group }}</a>
+            </li>
+          {% endfor %}
+        </ul>
+      {% else %}
+        {% blocktrans trimmed %}
+  <p class="font-weight-bold">You do not belong to any groups.</p>
```

Comment:
Rather put just the text into the translation, not the HTML as well. Less chance of it being incorrectly translated.

Reply:
changed